demo/kubernetes/rook-storageclass.yaml

```diff
@@ -19,8 +19,10 @@ metadata:
 provisioner: rook.io/block
 parameters:
   pool: replicapool
+  # The following specify the container image of the attacher
+  attacherImage: rook/toolbox:master
```

Comment:
I used this to state the container image that will do the attachment. I used toolbox because it was an image that already had `rookctl` which is what i am using to do the map/unmap.

We can, later on ,create a dedicated image for RBD attaching.